doc/src/configuration.md

```diff
@@ -1,10 +1,10 @@
 # Configuration
 
-Rustup has a settings file in [TOML](https://github.com/toml-lang/toml) format
-at `${RUSTUP_HOME}/settings.toml`. The schema for this file is not part of the
-public interface for rustup - the rustup CLI should be used to query and set
-settings.
+Rustup has a [TOML](https://github.com/toml-lang/toml) settings file at
+`${RUSTUP_HOME}/settings.toml` (which defaults to `~/.rustup` or
+`%USERPROFILE%/.rustup`). The schema for this file is not part of the public
+interface for rustup - the rustup CLI should be used to query and set settings.
 
 On Unix operating systems a fallback settings file is consulted for some
 settings. This fallback file is located at `/etc/rustup/settings.toml` and
-currently can define only `default_toolchain`.
+currently can only define `default_toolchain`.
```

Comment:
This is an incorrect change: https://dictionary.cambridge.org/grammar/british-grammar/only 
define is a verb, the section on focus and `only` as an adverb applies. The focus here is not the subject (`default_toolchain`), but the ability to define, so putting only in the mid position removes the focus from the subject.

Reply:
Thanks for the correction! I'll fix it.